src/diffusers/models/controlnet.py

```diff
@@ -24,19 +25,91 @@
 class ControlNetOutput(ControlNetOutput):
     def __init__(self, *args, **kwargs):
         deprecation_message = "Importing `ControlNetOutput` from `diffusers.models.controlnet` is deprecated and this will be removed in a future version. Please use `from diffusers.models.controlnets.controlnet import ControlNetOutput`, instead."
-        deprecate("ControlNetOutput", "0.34", deprecation_message)
```

Comment:
currently, you're getting a deprecation message like this - it is confusing because we are not deprecating `ControlNetModel`; we only deprecate importing from the previous file

```
FutureWarning: `ControlNetModel` is deprecated and will be removed in version 0.34. Importing `ControlNetModel` from `diffusers.models.controlnet` is deprecated and this will be removed in a future version. Please use `from diffusers.models.controlnets.controlnet import ControlNetModel`, instead.
  deprecate("ControlNetModel", "0.34", deprecation_message)
```